joern-cli/frontends/x2cpg/src/test/scala/io/joern/x2cpg/testfixtures/TestCpg.scala

```diff
@@ -26,6 +27,12 @@ abstract class TestCpg extends Cpg() with LanguageFrontend {
 
   protected def applyPasses(): Unit
 
+  private def applyPostProcessingPassesIfEnabled(): Unit = if (!_withPostProcessing) {
```

Comment:
I might be reading this wrong, but is the negated condition what you intended?

Reply:
Indeed this was the cause of the bugs that cropped up :D